### DISCUSSION TITLE
Way to refresh or refetch message?

### DISCUSSION THREAD

**Discussion by Sn0wCrack** [discord-net/Discord.Net#2362] [Q&A] [answered]
In the current implementation of my bot in Node using discord.js I'm able to easily refresh a message that was sent to my bot by using [discord.js' Message class' function called "fetch"
](https://github.com/discordjs/discord.js/blob/main/packages/discord.js/src/structures/Message.js#L836)

Is there any equivalent functionality in Discord.NET to refresh the data in a SocketUserMessage? 
I've tried looking around the documentation but can't seem to find anything that matches this functionality.

**Comment by csmir** [comment, marked as answer]
Socket entities do not need to be 'refreshed' because the instance served to you in events sources from cache. Any edits to the entity in this cache from Discord itself will automatically update the cached value internally (through the `MessageUpdated` event), and will thus already be updated at the moment its requested again. Keep in mind that the length of the message cache is controlled by `MessageCacheSize` in the `DiscordSocketConfig`

**Comment by csmir** [comment]
Please mark the discussion as answered if I've been able to help you out.
